projects/angular/src/accordion/accordion-panel.html

```diff
@@ -1,6 +1,10 @@
 <ng-container *ngIf="panel | async; let panel">
   <div [ngClass]="getPanelStateClasses(panel)">
-    <div class="clr-accordion-header">
+    <div
+      class="clr-accordion-header"
+      [attr.role]="headingEnabled || explicitHeadingLevel ? 'heading' : null"
+      [attr.aria-level]="headingEnabled || explicitHeadingLevel ? headingLevel : null"
```

Comment:
```suggestion
      [attr.role]="headingEnabled || (explicitHeadingLevel && headingEnabled !== false) ? 'heading' : null"
      [attr.aria-level]="headingEnabled || (explicitHeadingLevel && headingEnabled !== false) ? headingLevel : null"
```

I just realized I made a mistake before... I'm sorry this is difficult to get right. If the user sets an explicit heading level *and* disables the heading feature flag, it probably should not render the `role="heading"`.

Reply:
My thought process is that the explicit heading level implicitly enables the heading, but an explicitly disabling of the heading should take precendence.

Reply:
corrected

Reply:
Change reverted. It's not a bug, but a feature.

Providing `clrAccordionPanelHeadingEnabled=false` and `clrAccordionPanelHeadingLevel=3` is unsupported scenario.